docs/user/subprojects.rst

```diff
@@ -1,43 +1,67 @@
+For a main project ``example-project``, a subproject ``example-project-plugin`` can be made available as follows:
 
-Adding a subproject
--------------------
+* Main project: https://example-project.readthedocs.io/en/latest/
+* Subproject: https://example-project.readthedocs.io/projects/plugin/en/latest/
 
-In the admin dashboard for your project, select "Subprojects" from the menu.
-From this page you can add a subproject by typing in the project slug.
+.. seealso::
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+   :doc:`/guides/subprojects`
+     Learn how to create and manage subprojects
+
+   :doc:`/guides/intersphinx`
+     Learn how to use references between different Sphinx projects, for instance between subprojects
 
-You can use an alias for the subproject when it is created. This allows you to override the URL that is used to access it, giving more configurability to how you want to structure your projects.
 
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
+
+If the example project ``example-project`` was set up with a custom domain,
+``docs.example.com``, the URLs for projects ``example-project`` and ``example-project-plugin`` with alias ``plugin`` would
+respectively be at:
+
+* ``example-project``: https://docs.example.com/en/latest/
+* ``example-project-plugin``: https://docs.example.com/projects/plugin/en/latest/
+
+Using aliases
+-------------
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
+
+You can set your subproject's project name and :term:`slug` however you want,
+but we suggest prefixing it with the name of the main project.
 
-If the example project ``Foo`` was set up with a custom domain,
-``docs.example.com``, the URLs for projects ``Foo`` and ``Bar`` would
-respectively be at: https://docs.example.com/en/latest/ and
-https://docs.example.com/projects/bar/en/latest/
+Typically, a subproject is created with a ``$mainproject-`` prefix,
+for instance if the main project is called ``example-project-plugin`` and the subproject is called ``plugin``,
+then the subproject's Read the Docs project name will be ``example-project-plugin``.
+When adding the subproject,
+the alias is set to ``plugin`` and the project's URL becomes
+``example-project.readthedocs.io/projects/plugin``.
+
+When you add a subproject,
```

Comment:
```suggestion
When you add an alias to a subproject,
```

Reply:
Isn't the following part true regardless of adding an alias?

Reply:
Or do you mean because of the `/projects/plugin`?

I think we want to tell the reader mainly that the original domain `example-project-plugin.readthedocs.io` now redirects (regardless of the alias)

Reply:
Got it! In that case, I think we should merge this paragraph with the previous one since both talk about the same action "adding a subproject" and "where the project is gonna be hosted" now.

Reply:
Fused the two suggestions into a simpler one.